proto/pdpb.proto

```diff
@@ -103,33 +103,43 @@ service PD {
 }
 
 message WatchGlobalConfigRequest {
-
+    string config_path = 1;
+    int64 revision = 2;
 }
 
 message WatchGlobalConfigResponse {
     repeated GlobalConfigItem changes = 1;
+    int64 revision = 2;
 }
 
 message StoreGlobalConfigRequest {
-    repeated GlobalConfigItem changes = 1;
+    string config_path = 1;
+    repeated GlobalConfigItem changes = 2;
```

Comment:
breaking change, shouldn't change the field number

Reply:
This interface was previously used by TopSQL as a configuration center control switch, but then chose another way.
tikv/pd/tidb are not use it.